evap/static/js/sortable_query_binding.js

```diff
@@ -0,0 +1,61 @@
+/**
```

Comment:
i'm not sure what you mean, this is a lib just like the core sortable.js or bootstrap...?